### PR TITLE
XS✔ ◾ Co creation - improvements

### PR DESCRIPTION
**Pull request by bradystroud**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ Convo with @william-liebenberg 

> 2. What was changed?

✏️ Added some detail on how to do pair/mob programming

> 3. Did you do pair or mob programming (list names)?

✏️ @william-liebenberg

### CODE REVIEW

File: rules/co-creation-patterns/rule.md [hunk 1]
```diff
@@ -75,10 +75,17 @@ We can avoid this by using co-creation patterns. As a general rule, Pull Request
...
 
-Co-creation patterns can take several forms:
+Co-creation patterns can take some different forms:
 
-1. **Pair-programming**: Starting, reviewing and finishing a change together
-2. **Mob-programming**: Working in a small group, that collectively has all skills required
+1. **Pair-programming**: Two developers starting, reviewing and finishing a change together
+2. **Mob-programming**: Working in a small group, that collectively has all skills required. See <https://www.goretro.ai/post/mob-programming>
+
+For the patterns above, the similarities that there is a driver and a navigator.
```
Comment (danielmackay) [inline rules/co-creation-patterns/rule.md:83]: This doesn't sound right.  Can we fix up the grammar?

File: rules/co-creation-patterns/rule.md [hunk 2]
```diff
@@ -75,10 +75,17 @@ We can avoid this by using co-creation patterns. As a general rule, Pull Request
...
+
+**Driver** - Translate the ideas they are hearing from the navigator and turn them into code
+**Navigator(s)**  - Focus on the goal and instruct the driver how to get there, not micromanage the implementation
+
+These roles should swap often to keep a high level of focus and everyone an equal chance to participate as a driver and navigator.
```
Comment (danielmackay) [inline rules/co-creation-patterns/rule.md:88]:
```suggestion
These roles should swap often to keep a high level of focus and give everyone an equal chance to participate as a driver and navigator.
```

**Review by danielmackay — CHANGES_REQUESTED** [review]
Please fix

File: rules/co-creation-patterns/rule.md (hunk 1 above)
Comment (bradystroud) [reply rules/co-creation-patterns/rule.md]: done

**Review by danielmackay — APPROVED** [review]

**Pull request merged**